README.md

```diff
@@ -1,7 +1,9 @@
 # Usage
-Requires the `requests` module, which you can either install globally or in a virtualenv.
+Requires the `requests`, `pytz`, and `python-dateutil` modules, which you can either install globally or in a virtualenv.
 
 ```
+Install: pip install -r requirements.txt (expects Python to be installed along with pip)
```

Comment:
I'd rather this part be outside of the code block. :) The part inside is the exact output when run from command line.